dpe/src/commands/certify_key.rs

```diff
@@ -193,68 +196,25 @@ mod tests {
         format: CertifyKeyCmd::FORMAT_X509,
     };
 
-    #[test]
-    fn test_deserialize_certify_key() {
-        CfiCounter::reset_for_test();
-        let mut command = CommandHdr::new_for_test(Command::CERTIFY_KEY)
-            .as_bytes()
-            .to_vec();
-        command.extend(TEST_CERTIFY_KEY_CMD.as_bytes());
-        assert_eq!(
-            Ok(Command::CertifyKey(&TEST_CERTIFY_KEY_CMD)),
-            Command::deserialize(&command)
-        );
+    struct CertiKeyTestHelperArgs {
+        mark_dice_extensions_critical: bool,
     }
 
-    #[test]
-    fn test_certify_key_x509() {
+    fn certify_key_csr_test_helper(args: CertiKeyTestHelperArgs) {
```

Comment:
It looks like this is only called by one test. Should this just be moved to that test>

Reply:
Done!